### PR TITLE
chore: Replace deprecated Stack component with Flexbox and update tests for BannerRegularActions

### PR DESCRIPTION
**Pull request by avirati**
No visual changes expected

Before : 
Screenshot 2023-09-07 at 07 37 35

After :
Screenshot 2023-09-07 at 07 31 20

### CODE REVIEW

**Review by andoulla — APPROVED** [review]

File: lib/src/components/banner/banner-regular/BannerRegularActions.tsx [hunk 1]
```diff
@@ -3,34 +3,51 @@ import * as React from 'react'
...
   )
 
   return (
-    <Stack gap={gap} {...props}>
+    <Flex
+      css={{
+        gap,
+        flexDirection,
+        ...css
+      }}
```
Comment (thomasdigby) [inline lib/src/components/banner/banner-regular/BannerRegularActions.tsx:48]: You can do similar things with `Flex` now that you used to be able to do only on `Stack` https://atomlearning.design/components/layout/flex. That will help to simplify the CSS stuff here

```jsx
<Flex gap={gap} direction={flexDirection} {...props}>
```

Comment (thomasdigby) [reply lib/src/components/banner/banner-regular/BannerRegularActions.tsx:48]: You'll need to revert your `toGap` changes along with this

**Comment by thomasdigby** [comment]
Worth keeping in mind that `Flex` doesn't provide a default `justifyContent`, `alignItems` or `flexWrap` like `Stack` used to, so you should test a few responsive layouts to confirm that it still works as expected

File: lib/src/components/banner/banner-regular/BannerRegularActions.tsx (hunk 1 above)
Comment (avirati) [reply lib/src/components/banner/banner-regular/BannerRegularActions.tsx:48]: Makes sense, thanks

Comment (avirati) [reply lib/src/components/banner/banner-regular/BannerRegularActions.tsx]: Implemented the changes 👍

**Comment by thomasdigby** [comment]
> While developing this component, we decided that this component will not depend on css breakpoints but instead, use `size` prop to decide how it would look. Do you think we still need to address these ?

Yeah that's fine, you just need to be aware that it will no longer align, justify, and wrap content like it used. It might be fine, but maybe more content might trigger a layout regression

**Review by thomasdigby — APPROVED** [review]

**Pull request merged**